CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#11158](https://github.com/CocoaPods/CocoaPods/pull/11158)
 
+* cdn_url? check shouldn't crash operations. Catch YAML.load failure  
```

Comment:
@KostyaSha Perhaps you could phrase it more like "Add catch for YAML syntax error to prevent crash.  "

Reply:
changed, but keywords needs to match actual error so people can google better